android-sdk/android/src/main/kotlin/io/logto/sdk/android/util/LogtoUtils.kt

```diff
@@ -1,11 +1,12 @@
 package io.logto.sdk.android.util
 
+import java.util.Calendar
 import kotlin.math.floor
 
 object LogtoUtils {
     private const val MILLIS_PER_SECOND = 1000L
 
-    fun nowRoundToSec() = floor((System.currentTimeMillis() / MILLIS_PER_SECOND).toDouble()).toLong()
+    fun nowRoundToSec() = floor((Calendar.getInstance().timeInMillis / MILLIS_PER_SECOND).toDouble()).toLong()
```

Comment:
Just want to confirm the idea behind this update. Since not mentioned in the PR summary

Reply:
We can not mock the `System`, so use `Calendar` instead.
https://github.com/mockk/mockk/issues/55